scapy/contrib/nsh.py

```diff
@@ -36,7 +36,6 @@ class NSHTLV(Packet):
         PacketListField('Metadata', None, XIntField, count_from='Len')
     ]
 
-
```

Comment:
That's nothing but please try to avoid altering lines when you're not changing the code, as it breaks the history for no reason. Plus in this case I think PEP-8 recommends two blank lines.